tests/macsec/test_macsec.py

```diff
@@ -79,10 +82,27 @@ def _test_mka_session():
 
 
 class TestDataPlane():
-    BATCH_COUNT = 100
+    BATCH_COUNT = 10
+
+    def test_server_to_eos_neighbor(self, duthost, ctrl_links, downstream_links, upstream_links, nbr_device_numbers, ptfadapter):
+        ptfadapter.dataplane.set_qlen(TestDataPlane.BATCH_COUNT * 30)
+        down_port, down_link = downstream_links.items()[0]
+        for ctrl_port in ctrl_links.keys():
+            up_link = upstream_links[ctrl_port]
+            dut_macaddress = duthost.get_dut_iface_mac(ctrl_port)
+            payload = "{} -> {}".format(down_link["name"], up_link["name"])
+            logging.info(payload)
+            # Source mac address is not useful in this test case and we use an arbitrary mac address as the source
+            pkt = create_pkt(
+                "00:01:02:03:04:05", dut_macaddress, "1.2.3.4", up_link["ipv4_addr"], bytes(payload))
+            exp_pkt = create_exp_pkt(pkt, pkt[scapy.IP].ttl - 1)
+            testutils.send_packet(
+                ptfadapter, down_link["ptf_port_id"], pkt, TestDataPlane.BATCH_COUNT)
+            check_macsec_pkt(macsec_attr=get_macsec_attr(duthost, ctrl_port), test=ptfadapter,
+                             ptf_port_id=up_link["ptf_port_id"],  exp_pkt=exp_pkt, timeout=10)
```

Comment:
Remove. Update it in multiple members of portchannel

Reply:
Ok. Do it commonly while multiple members of portchannel for both vsonic and ceos neighbors.